packages/core/src/external/carequality/ihe-gateway-v2/schema.ts

```diff
@@ -89,42 +89,29 @@ export const samlHeaderSchema = z.object({
     }),
     Assertion: z.object({
       AttributeStatement: z.object({
-        Attribute: z.tuple([
-          z.object({
-            _Name: z.literal("urn:oasis:names:tc:xspa:1.0:subject:subject-id"),
-            _NameFormat: z.string().optional(),
-            AttributeValue: textSchema,
-          }),
-          z.object({
-            _Name: z.literal("urn:oasis:names:tc:xspa:1.0:subject:organization"),
-            _NameFormat: z.string().optional(),
-            AttributeValue: textSchema,
-          }),
-          z.object({
-            _Name: z.literal("urn:oasis:names:tc:xspa:1.0:subject:organization-id"),
-            _NameFormat: z.string().optional(),
-            AttributeValue: textSchema,
-          }),
-          z.object({
-            _Name: z.literal("urn:nhin:names:saml:homeCommunityId"),
-            _NameFormat: z.string().optional(),
-            AttributeValue: textSchema,
-          }),
-          z.object({
-            _Name: z.literal("urn:oasis:names:tc:xacml:2.0:subject:role"),
-            _NameFormat: z.string().optional(),
-            AttributeValue: z.object({
-              Role: codeSchema,
+        Attribute: z.array(
```

Comment:
we had this error
`  
{
    "code": "too_big",
    "maximum": 6,
    "inclusive": true,
    "exact": false,
    "type": "array",
    "path": [
        "Envelope",
        "Header",
        "Security",
        "Assertion",
        "AttributeStatement",
        "Attribute"
    ],
    "message": "Array must contain at most 6 element(s)"
}`

since we were requiring those specific 6 attributes. Now we are accepting an array of however many attributes that can conform to any of these three types